### DISCUSSION TITLE
[Bug] [unknown source] [scrcpy] [Extended Display] connecting to scrcpy turns phone to tablet

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#621] [General]
ok, so this may sound weird, but what I mean is that when I connect to my phone with scrcpy and use virtual extended display, my phone just decides it's a tablet and shows the nav bar parallel to the status bar in landscape mode, and doesn't go away until I restart my phone. I don't know if this is a bug with Android, Samsung, or ya-webadb is doing it.

**Comment by yume-chan** [comment]
Probably a ROM-specific thing. Can you add a screenshot?

**Comment by CorruptionVR** [comment]
![Screenshot_20240108_202830_Discord](https://github.com/yume-chan/ya-webadb/assets/78228404/baa2ef80-ac72-4a7b-b9a4-134f94d27f0d)
here it is
don't mind the chats

**Reply by yume-chan** [reply]
Why does it look like Windows Phone?

**Reply by CorruptionVR** [reply]
Oh, it's a samsung, and I'm using navstar. I like the look of wp, so, I recreated it

**Reply by yume-chan** [reply]
So you always have the navigation bar on, but normally it's on the shorter side of your screen (left or right when in horizontal), but after creating a virtual display, the navigation bar is always on the bottom?

I decompiled navstar, it determines the location of navigation bar by whether the device is a tablet and current screen orientation. However, whether the device is a tablet is determined by the system and I can't find the code for that. It may exist in a system jar package, or a system app package, so it's hard to find.

There might not be anything more I can do at the moment until I get a Samsung device to test.
